docs/api/associations/belongs-to-many.md

```diff
@@ -10,8 +10,8 @@ When the join table has additional attributes, these can be passed in the option
 UserProject = sequelize.define('user_project', {
   role: Sequelize.STRING
 });
-User.belongsToMany(Project, { through: UserProject });
-Project.belongsToMany(User, { through: UserProject });
+User.belongsToMany(Project, { through: 'UserProject' });
+Project.belongsToMany(User, { through: 'UserProject' });
```

Comment:
It doesn't work for me.

Also the `allowNull` option isn't taken into account when associated with a `name` attribute:

```
        Conversation.belongsToMany(models.User, {
          through: 'UserConversations',
          foreignKey: {
            name: 'UserId',
            allowNull: false
          }
        });
```

Without the name attribute, the foreignKey name is set to 'undefined'.